recipes/ocetrac/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "ocetrac" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: ocetrac-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e9f032ca20ac81f5423c8f6fe1d6b969b675a5762068d9aa44a731416fd5da43
+  
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+  
+requirements:
+  build:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - pytest
```

Comment:
I don't think `pytest` is actually needed to use `ocetrac`?

Reply:
It's used in the tests, but you're right, you don't need it to use `ocetrac`.